src/Protocol/ActivityPub/Receiver.php

```diff
@@ -1076,14 +1079,15 @@ public static function getReceiverURL(array $activity): array
 	 * Fetch the receiver list from an activity array
 	 *
 	 * @param array   $activity
-	 * @param string  $actor
-	 * @param array   $tags
-	 * @param boolean $fetch_unlisted
+	 * @param string $actor
+	 * @param array  $tags
+	 * @param bool   $fetch_unlisted
+	 * @param bool   $push
```

Comment:
Can you please explicit the use of this flag?

Reply:
The flag `$push` is passed through the several functions of the receiver and indicates that the post had been pushed to the system and is not fetched by the system. This is important since that indicates that the sender is online and should be reachable to fetch newest contact data from the actor.

Reply:
Thank you, I meant in the code but I guess this will do for now.